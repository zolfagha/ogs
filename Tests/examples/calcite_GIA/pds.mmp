GeoSys-MMP  Material Medium Properties ------------------------------------
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  1
 $GEOMETRY_AREA
  1.000000e+000
 $POROSITY
  2  0.32
 $TORTUOSITY
  1   1.000000e+000
 $PERMEABILITY_TENSOR
  ISOTROPIC  1.13386e-5
 $PERMEABILITY_FUNCTION_POROSITY
  1 0
 $MASS_DISPERSION
  1  0.0067  0.0 
 $DENSITY
  1  1800.0
 #STOP

 




#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  1
 $GEOMETRY_AREA
  1.000000e+000
 $POROSITY
  1  0.32
 $TORTUOSITY
  1   1.000000e+000
 $PERMEABILITY_TENSOR
  ISOTROPIC  1.13386e-5  original value
 $MASS_DISPERSION
  1  0.0067  0.0 
 $DENSITY
  1  1800.0






 
 
 1  0.0067  0.1000
#SOIL_PROPERTIES
 1  1.000000e+000             ; dimension, (area), 
 0                            ; porosity model
 0.32  1.000000e+000          ; porosity, tortuosity
 0                            ; mobile immobile model
 0                            ; lithological component
 0                            ; maximum sorption model
 0 ; nonlinear flow parameter
 0.000000e+000 ; storativity
 0 0  1.157000e-012 ; permeability model, permeability tensor, permeabilities
 0  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000 ; k-S function
 0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000  0.000000e+000 ; p-S function
; 0.100000e+000  0.000000e+000 ; mass dispersion parameters
 0.00670000e+000  0.000000e+000 ; mass dispersion parameters
 0.000000e+000  0.000000e+000 ; heat dispersion parameters
 1.800000e+003  0.000000e+000 ; rock density, heat capacity
 0  0.000000e+000 ; heat conductivity parameters


$MASS_DISPERSION
  1  0.25  0.0
 $TORTUOSITY
  1   1.000000e+000
