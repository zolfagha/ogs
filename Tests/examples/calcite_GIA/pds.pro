<?xml version="1.0"?>
<ogs6>
<processParameters>
	<GROUNDWATER_FLOW TimeGroupID="0"/>
	<HEAD_TO_ELEMENT_VELOCITY TimeGroupID="0"/>
    <REACT_GIA TimeGroupID="0"/>
</processParameters>
<coupling>
    <P algorithm="Serial" convergence="FemFunctionConvergenceCheck" max_itr="15" epsilon="1e-4">
        <out>HEAD</out>
        <out>VELOCITY</out>
        <out>Tracer</out>
        <problems>
            <M name="GROUNDWATER_FLOW" type="GROUNDWATER_FLOW">
                <out>HEAD</out>
            </M>
            <M name="HEAD_TO_ELEMENT_VELOCITY" type="HEAD_TO_ELEMENT_VELOCITY">
                <in>HEAD</in>
                <out>VELOCITY</out>
            </M>
            <M name="REACT_GIA" type="REACT_GIA">
                <in>VELOCITY</in>
                <out>Tracer</out>
            </M>
        </problems>
    </P>
</coupling>
</ogs6> 

