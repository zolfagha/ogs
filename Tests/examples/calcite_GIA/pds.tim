GeoSys-TIM: Timestepping ------------------------------------------------
#TIME_STEPPING
 $PCS_TYPE
  REACT_GIA
 $TIME_END
  21000.0
 $TIME_START
  0.0
$TIME_CONTROL
   NEWTON_ADAPTIVE
 MAX_TIME_STEP
     300
 MIN_TIME_STEP
     20.0
 ITER_TIMES_AND_MULTIPLYER
   4
   0    1.6
   3    1.0
   5    0.5
   7    0.25  
#STOP

#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   1    20.0
   1    20980.0
 $TIME_END
  21000.0
 $TIME_START
  0.0

   4
   0    1.6
   7    1.0
   15   0.5
   35   0.25   

   4
   0    1.6
   3    1.0
   5    0.5
   7    0.25  

#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   1    10.0
   1    20990.0
 $TIME_END
  21000.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  REACT_GIA
 $TIME_END
  21000.0
 $TIME_START
  0.0
$TIME_CONTROL
   NEWTON_ADAPTIVE
 MAX_TIME_STEP
     1000
 MIN_TIME_STEP
     10.0
 ITER_TIMES_AND_MULTIPLYER
   4
   0    1.6
   7    1.0
   15   0.5
   20   0.25 



#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   210  100.0
 $TIME_END
  21000.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  REACT_GIA
 $TIME_STEPS
   210  100.0
 $TIME_END
  21000.0
 $TIME_START
  0.0
