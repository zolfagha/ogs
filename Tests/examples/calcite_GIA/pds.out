GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  H+
  Ca2+
  Mg2+
  CO32-
  Cl-
  HCO3-
  Calcite
  Dolomite(dis)
 $ELE_VALUES
  POROSITY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP

